cumulusci/robotframework/pageobjects/ObjectManagerPageObject.py

```diff
@@ -28,13 +28,42 @@ class ObjectManagerPage(BasePage):
         | Go to page   ObjectManager  Contact
         """
 
+    global search_button
```

Comment:
You can delete all of these `global` statements if you move the definition of the variables outside of the class. For example:

```
object_manager = {
    "button": "//input[@title='{}']",
   ...
}
search_button = object_manager["input"].format("globalQuickfind")
currency_locator = object_manager["input"].format("dtypeC")

```

However, I don't see the point of both having the locators in a dictionary and then creating variables from the dictionary. Instead of creating the dictionary you can just define the shortcuts to have the xpath and  eliminate the need of having both the locators and shortcuts.

```
search_button = "//input[@title='globalQuickfind']"
currency_locator = "////input[@id='dtypeC']"
...
```
This might be a good subject to talk about in one of the robot checkins - what is the preferred way to reference locators in a page object? Maybe your solution of using individual variables is good. Though, if we ever make it easier to re-use locators between page objects, having a single dictionary would make that easier. 

I don't want this issue to prevent this code from getting merged, but at the very least you should remove all of the global statements, and then move the definition of the variables outside of the class.

Reply:
For now, changed as per your suggestion